tests/lab/test_lab_value_common.py

```
@@ -0,0 +1,422 @@
+from django.test import TestCase
+from logging import getLogger
+
+from endoreg_db.models import (
+    LabValue,
+)
+from endoreg_db.models.medical.laboratory.lab_value import CommonLabValues # Keep for class structure
```

Pylint (3.3.7)

[error] Cannot import 'endoreg_db.models.medical.laboratory.lab_value' due to 'invalid syntax (endoreg_db.models.medical.laboratory.lab_value, line 132)'

(E0001)